index.js

```diff
@@ -1,3 +1,6 @@
+document.getElementById('end_screen').style.display = 'none'
+document.getElementById('game_screen').style.display = 'none'
```

Comment:
Since these default to none, I would suggest setting it in the CSS rather than JS just because it's more efficient